### PR TITLE
Fix/ru.onlinemschool.com #31817

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/31817

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -2368,7 +2368,11 @@ heroero.com##.block-video > .table > .opt
 ||heroero.com/temp/2ava.jpg
 efukt.com##.efukt-widget-slider-nice-try-adblockers
 sportnews.to##span.mol-ads-label-container
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=skylinewebcams.com,important
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=skylinewebcams.com|onlinemschool.com,important
```
Comment (BlazDT) [inline EnglishFilter/sections/specific.txt]: According to whitelist comment blocking that element causes infinite loading. If no infinite loading happens with that rule, please remove whitelist rule.

Left-over for desktop:
#oms_res_block  > div[id^="oms_oag"][style="text-align:center;margin:1em 0;"]

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Review by BlazDT — APPROVED** [review]

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -2368,7 +2368,12 @@ heroero.com##.block-video > .table > .opt
...
+!+ PLATFORM(android, ios)
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt]: do you mean `AG for Android` or Content Blocker?

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -2368,7 +2368,12 @@ heroero.com##.block-video > .table > .opt
 ||heroero.com/temp/2ava.jpg
 efukt.com##.efukt-widget-slider-nice-try-adblockers
 sportnews.to##span.mol-ads-label-container
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=skylinewebcams.com,important
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=skylinewebcams.com|onlinemschool.com,important
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt]: without exclusion onlinemschool.com is unnecessary

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/specific.txt]: For android

File: MobileFilter/sections/specific_web.txt [hunk 4]
```diff
@@ -48,6 +48,8 @@ vg247.com##div[id^="mobile-mpu-"]
 woman.ru##.mob-in__container
 woman.ru##.notification__ad
 woman.ru##div[data-name="MobListVideo"]
+onlinemschool.com###oms_res_header
+onlinemschool.com##oms_share_s
```
Comment (BlazDT) [inline MobileFilter/sections/specific_web.txt]: To Social filter, also missing # for the ID `oms_share_s`

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt [hunk 5]
```diff
@@ -54,6 +54,8 @@ fishki.net##.content__figure__share__social
 womanadvice.ru###mobile_social
 gazzetta.gr##.field-name-bottom-share-buttons
 bursahayat.com.tr##.share-bar > a
+onlinemschool.com###oms_res_header
+onlinemschool.com###oms_share_s
```
Comment (BlazDT) [inline SocialFilter/sections/specific.txt]: I meant only the share rule to Social, not the other one.

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Review by BlazDT — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**